docs/_data/docstoc.yml

```diff
@@ -8,39 +8,39 @@
       children:
       - title: 'Installing Codewind for VS Code'
         url: mdt-vsc-getting-started.html
-      - title: 'Creating your first project'
+      - title: 'Creating your first VS Code Codewind project'
         url: mdt-vsc-firstproject.html
-      - title: 'Making a code change'
+      - title: 'Making a code change with VS Code'
         url: vsc-codechange.html
     - title: 'Eclipse'
       children:
       - title: 'Installing Codewind for Eclipse'
         url: mdt-eclipse-getting-started.html
-      - title: 'Creating your first project'
+      - title: 'Creating your first Eclipse Codewind project'
         url: mdt-eclipse-firstproject.html
-      - title: 'Making a code change'
+      - title: 'Making a code change with Eclipse'
         url: mdt-eclipse-codechange.html
     - title: 'Eclipse Che'
       children:
       - title: 'Installing Eclipse Che for Codewind'
         url: mdt-che-installinfo.html
-      - title: 'Adding registries in Che'
+      - title: 'Adding a container registry in Codewind'
         url: mdt-che-setupregistries.html
       - title: 'Creating a Codewind workspace in Che'
         url: mdt-che-createcodewindworkspace.html
       - title: 'Creating your first project with Codewind for Eclipse Che'
         url: mdt-che-createfirstproject.html
-      - title: 'Setup to use Tekton Pipelines'
+      - title: 'Configuring Codewind for Tekton pipelines'
         url: mdt-che-tektonpipelines.html
-      - title: 'Using Codewind in Theia'
+      - title: 'Using Codewind from the Che Theia IDE'
         url: mdt-che-usingtheia.html
       - title: 'OpenShift Do (odo) support in Codewind'
         url: mdt-che-odo-support.html         
-      - title: 'Using the OpenShift Registry'
+      - title: 'Adding the OpenShift internal registry with Codewind'
         url: openshiftregistry.html
       - title: 'Support for multiple users'
         url: supportingmultipleusers.html
-      - title: 'Installing Kibana and filtering logs in IBM Cloud Private'
+      - title: 'Viewing logs using Kibana in IBM Cloud Private'
         url: viewkibanalogs.html
       - title: 'Uninstalling Codewind for Eclipse Che'
         url: mdt-che-uninstall.html
```

Comment:
I don't see the IntelliJ docs listed here in this TOC

Reply:
see IntelliJ here: http://9.42.27.84:4321/codewind/mdt-intellij-getting-started.html